openreview/tools.py

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
+from __future__ import print_function
 import openreview
 import re
 import datetime
 
 super_user_id = 'OpenReview.net'
 
-def get_profile(client, value):
-    profile = None
-    try:
-        profile = client.get_profile(value)
-    except openreview.OpenReviewException as e:
-        # throw an error if it is something other than "not found"
-        if e[0][0] != 'Profile not found':
-            print "OpenReviewException: {0}".format(e)
-            return e
-    return profile
-
 def create_profile(client, email, first, last, middle = None, allow_duplicates = False):
 
+    '''
+    Given email, first name, last name, and middle name (optional), creates and returns
+    a user profile.
+
+    If a profile with the same name exists, and allow_duplicates is False, an exception is raised.
+
+    If a profile with the same name exists and allow_duplicates is True, a profile is created with
+    the next largest number (e.g. if ~Michael_Spector1 exists, ~Michael_Spector2 will be created)
+    '''
+
     profile = get_profile(client, email)
```

Comment:
we can remove the get_profile function because it is being used here, could you revert that change too?

Reply:
do you mean "can't" remove? so you want me to get_profile back into tools?

Reply:
I'm sorry, can't.